components/Link/hooks.ts

```diff
@@ -31,7 +31,19 @@ export const useNormalizedHref = (href: string) => {
     ? href.substring(basePath.length)
     : href;
 
-  const { scope } = useContext(DocsContext);
+  const scopes = {
+    oss: true,
+    cloud: true,
+    enterprise: true,
+  };
+
+  let scope: "oss" | "cloud" | "enterprise" = useContext(DocsContext).scope;
```

Comment:
FYI: We already have types for scopes and the array of available scopes you can use here instead of defining them again.

See: https://github.com/gravitational/docs/blob/main/layouts/DocsPage/types.ts#L6-L9

Reply:
Thanks! Fixed this in 182adf6